lib/ansible/modules/apt_key.py

```diff
@@ -267,7 +274,6 @@ def main():
             url=dict(type='str'),
             data=dict(type='str'),
             file=dict(type='path'),
-            key=dict(type='str'),
```

Comment:
Options should not be removed without a deprecation cycle.

Reply:
@felixfontein Could you please check https://github.com/ansible/ansible/pull/70319/commits/cfcacaa1cf4d98fad68120b270b545bf0148fbb4 and these minor fixes in https://github.com/ansible/ansible/pull/70319/commits/c7c3a51e81761df7e94db0dc307f74627e0d2851 and https://github.com/ansible/ansible/pull/70319/commits/ea20be4462c4bd8d47082fa433e425f115990d8a?